fea-rs/src/compile/compile_ctx.rs

```diff
@@ -2222,4 +2238,21 @@ mod tests {
             ]
         );
     }
+
+    fn parse_name_spec(fea: &str) -> crate::compile::tables::NameSpec {
+        resolve_name_spec(&typed::NameSpec {
+            inner: crate::parse::parse_string(fea).0.root,
+        })
+    }
+
+    #[test]
+    fn parse_name_spec_drop_quotes() {
+        assert_eq!(
+            vec!["", "duck"],
+            vec![
+                parse_name_spec("3 1 0x409 \"\"").string,
+                parse_name_spec("3 1 0x409 \"duck\"").string
+            ]
```

Comment:
trivia, but fwiw you don't need the `vec` here you can just have these be arrays..